frontend/packages/dev-console/src/components/AddPage.tsx

```diff
@@ -12,11 +14,94 @@ export interface AddPageProps {
     ns?: string;
   }>;
 }
+
+interface ResourcesType {
+  deploymentConfigs: K8sResourceKind;
+  deployments: K8sResourceKind;
+  daemonSets: K8sResourceKind;
+  statefulSets: K8sResourceKind;
+}
+interface EmptyStateLoaderProps {
+  resources?: ResourcesType;
+  loaded?: boolean;
+  loadError?: string;
+}
+
 const openProjectModal = () =>
   createProjectModal({
     blocking: true,
     onSubmit: (project) => history.push(`/add/ns/${project.metadata.name}`),
   });
+
+const EmptyStateLoader: React.FC<EmptyStateLoaderProps> = ({ resources, loaded, loadError }) => {
+  const [isWorkLoadsPresent, setIsWorkLoadsPresent] = React.useState(false);
+
+  React.useEffect(() => {
+    if (loaded) {
+      setIsWorkLoadsPresent(
+        _.isEmpty(resources.deploymentConfigs.data) &&
+          _.isEmpty(resources.deployments.data) &&
+          _.isEmpty(resources.daemonSets.data) &&
+          _.isEmpty(resources.statefulSets.data),
```

Comment:
So right now `isWorkloadsPresent` will be false when there are workloads and true when there are none. It will make more sense if it is the other way around.

Reply:
@vikram-raj please address. Logic and naming seem to contradict as the it's inverted.

Reply:
Sorry, I missed the review comment from Divynashi I will address this.

Reply:
I changed ```isWorkloadsPresent``` to ```noWorkloads```. Now this condition more sense.